Dockerfile.test

```
@@ -1,9 +1,14 @@
 FROM python:3.13
+
 COPY --from=ghcr.io/astral-sh/uv:latest /uv /bin/uv
 
 ENV PYTEST_ADDOPTS="--rootdir=/tests"
 
 WORKDIR /tests
 
+# copied separately from volume definitions
 COPY pyproject.toml uv.lock .
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)